types/urllib/README.md

```diff
@@ -0,0 +1,29 @@
+# DefinitelyTyped for Urllib
```

Comment:
A custom `README` won't be published. If there's something important to include this could go in a comment in `index.d.ts`. But in this case, most of this info is available elsewhere already, and people can go to the package's homepage to get usage examples.